src/js/components/PeaksList/Peak.jsx

```diff
@@ -3,9 +3,10 @@ import { Link } from 'react-router-dom';
 import styles from './Peak.module.css';
 
 const Peak = ({ peak }) => {
+  // TODO change for proper address
+  const id = peak._links.self.href.split('http://localhost:8080/api/peaks/')[1];
```

Comment:
Rozumiem, że jest to TODO, ale czemu tworzysz całą strukturę _links.self.href...

Reply:
Żeby wyciągnąć id szczytu, bo nie wiem czy jest bezpośrednio pod peak.id

Reply:
Raczej dostaniemy to z backendu pod z kluczem id, a wpp dodamy to pole w hooku, który zaciągnie dane z backendu

Reply:
Ale zmienię na razie żeby było